### ISSUE TITLE
Builds fail when using alpha/beta versions

### ISSUE THREAD

**Issue by jeffsawatzky** [cjolowicz/cookiecutter-hypermodern-python#473]
I used poetry to create a prerelease, so my pypackage.toml looks something like this:
```
[tool.poetry]
name = "my-project"
version = "0.0.1-alpha.0"
```

When I run something such as `nox -d docs` I get the following output
```
nox -s docs
nox > Running session docs
nox > Creating virtual environment (virtualenv) using python3.8 in .nox/docs
nox > poetry export --format=requirements.txt --output=/var/folders/9z/jbvh01ds3zqcphy_8ytw10680000gp/T/tmp30pogygj/requirements.txt
nox > pip install --requirement=/var/folders/9z/jbvh01ds3zqcphy_8ytw10680000gp/T/tmp30pogygj/requirements.txt
nox > poetry build --format=wheel
Building my-project (0.0.1-alpha.0)
 - Building wheel
 - Built my_project-0.0.1a0-py3-none-any.whl
nox > poetry version
nox > pip install --no-deps --force-reinstall dist/my_project-0.0.1-alpha.0-py3-none-any.whl
nox > Command pip install --no-deps --force-reinstall dist/my_project-0.0.1-alpha.0-py3-none-any.whl failed with exit code 1:
WARNING: Requirement 'dist/my_project-0.0.1-alpha.0-py3-none-any.whl' looks like a filename, but the file does not exist
ERROR: my_project-0.0.1-alpha.0-py3-none-any.whl is not a supported wheel on this platform.
WARNING: You are using pip version 20.1.1; however, version 20.2 is available.
You should consider upgrading via the '/Users/jsawatzky/Code/private/my-project/.nox/docs/bin/python -m pip install --upgrade pip' command.
nox > Session docs failed.
```

So poetry is building the wheel with the name:
`my_project-0.0.1a0-py3-none-any.whl`
but the noxfile.py tries to use:
`my_project-0.0.1-alpha.0-py3-none-any.whl`

Am I doing something wrong?

**Comment by cjolowicz** [comment]
No, this is a bug in the template. Thanks for reporting this. 👍 There is already an issue open for this in #740 so I'm closing this one as a duplicate.

**Comment by cjolowicz** [comment]
Hi @jeffsawatzky this was fixed in #487
